src/store.js

```diff
@@ -33,32 +33,32 @@ export class Store {
         passwordVerify: '',
         seedVerify: ['', '', ''],
       },
-      transactions: null,
+      transactions: [],
```

Comment:
these look great! interesting we weren't initializing with empty lists from the beginning :O